docs/oauth2-examples-azure.md

```diff
@@ -21,11 +21,10 @@ limitations under the License.
 
 # Use Azure Active Directory (Azure AD) as OAuth 2.0 server
 
-Let's test the following 3 OAuth flows:
+Demonstrate how to authenticate using OAuth 2.0 protocol
+and Azure Active Directory as Authorization Server using the following flows:
 
-* Access management UI via a browser :ballot_box_with_check:
-* Access management rest api :construction:
-* Access AMQP protocol :construction:
+* Access management UI via a browser
 
```

Comment:
Add "the" after "Access" Also in these headings would it be good to add "RabbitMQ" before "Management UI" in all/some instances when referencing the "Management UI" just from the perspective of google search results, it would be clear that these is the "RabbitMQ Management UI" and not any other product Management UI ?